src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -18,10 +18,10 @@ public abstract class AddCommand extends Command {
 
     public static final String COMMAND_WORD = "add";
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Adds an entity (person or appointment) to the address book. "
+            + ": Adds an entity to address or appointment book. \n"
             + "Parameters: "
-            + "ENTITY_TYPE "
-            + "ENTITY_ARGUMENTS..."
+            + "ENTITY_TYPE (person/appt) "
+            + "DATA_FIELDS [MORE_DATA_FIELDS]... \n"
             + "Example: " + COMMAND_WORD + " person "
```

Comment:
Should we use `ParserUtil.PERSON_ENTITY_STRING` instead of coding `person` manually?

Reply:
OK, I'll make the refactor for this.